cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -447,7 +446,7 @@ getTxBody (ShelleyTx era tx) =
   where
     getShelleyTxBody :: forall ledgerera.
```

Comment:
This shouldn't need `forall ledgerera.`

Reply:
It's not necessary to be explicit about the forall here, but it is logically a forall for `ledgerera` here, whereas the use of `era` is from the outer scope. I think it's clearer to be explicit about that.

When our types are simple it makes sense to let the foralls be implicit, but once it gets complicated then I think it is clearer with explicit foralls.